### PR TITLE
Add CHECKOUT_ID to Realex and make ORDER_ID unique

### PR DESCRIPTION
**Pull request by andrewpaliga**
- As per discussion with Realex
- Realex requires the ORDER_ID to be unique for every transaction (including fails)
- Realex will pass shopify checkout_id in "CHECKOUT_ID" field

@girasquid @davidjrice

### CODE REVIEW

File: test/unit/integrations/realex/realex_module_test.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class RealexModuleTest < Test::Unit::TestCase
   include OffsitePayments::Integrations
 
   def test_helper_method
-    assert_instance_of Realex::Helper, Realex.helper(123, 'test')
+    assert_instance_of Realex::Helper, Realex.helper('123', 'test')
```
Comment (girasquid) [inline test/unit/integrations/realex/realex_module_test.rb]: Why this change?

**Comment by girasquid** [comment]
One question, otherwise looks ok.

File: test/unit/integrations/realex/realex_module_test.rb (hunk 1 above)
Comment (andrewpaliga) [inline test/unit/integrations/realex/realex_module_test.rb]: You need to pass in a string or joining the `order + @timestamp` throws an exception

Comment (girasquid) [inline test/unit/integrations/realex/realex_module_test.rb]: :cool:
